.github/workflows/latest-bitcoind.yml

```diff
@@ -0,0 +1,51 @@
+name: Latest Bitcoin Core
+
+on:
+  schedule:
+    # Run at midnight on Sunday and Wednesday.
+    - cron: '0 0 * * 0,3'
+
+jobs:
+
+  regression-tests:
+    runs-on: ubuntu-latest
+    timeout-minutes: 90
+    steps:
+      - name: Checkout bitcoind master
+        uses: actions/checkout@v2
+        with:
+          repository: bitcoin/bitcoin
+          path: bitcoin
+
+      - name: Install bitcoind dependencies
+        run: sudo apt-get install build-essential libtool autotools-dev automake pkg-config bsdmainutils python3 libssl-dev libevent-dev libboost-system-dev libboost-filesystem-dev libboost-chrono-dev libboost-test-dev libboost-thread-dev libminiupnpc-dev libzmq3-dev libqt5gui5 libqt5core5a libqt5dbus5 qttools5-dev qttools5-dev-tools libprotobuf-dev protobuf-compiler git libsqlite3-dev ccache
+        working-directory: ./bitcoin
+
+      - name: Autogen bitcoind
+        run: ./autogen.sh
+        working-directory: ./bitcoin
+
+      - name: Configure bitcoind
+        run: ./configure
```

Comment:
some options that should make this faster

`--with-zmq --without-gui --disable-shared --with-pic --disable-tests --disable-bench`

Reply:
Thanks, good idea!

Reply:
Thanks for the tip, done in https://github.com/ACINQ/eclair/pull/2027/commits/21efe18b3202dfd9e807a6b78f02baa6b7bfc770